doc/specs/#476 - Pinned packages.md

```diff
@@ -0,0 +1,180 @@
+---
+author: Easton Pillay jedieaston/easton@planeteaston.com
+created on: 2022-01-25
+last updated: 2022-02-02
+issue id: 476
+---
+
+# Pinning a package
+
+For [#476](https://github.com/microsoft/winget-cli/issues/476)
+
+## Abstract
+
+This spec describes the functionality behind "pinning" a package (freezing a package at a certain version so that it will be not be automatically upgraded).
```

Comment:
I'd clarify that it wouldn't be automatically upgraded via `winget upgrade --all`.

Reply:
Resolved!